feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -203,6 +204,15 @@ class BillingWrapper(
         replaceSkuInfo: ReplaceSkuInfo?,
         presentedOfferingIdentifier: String?
     ) {
+        val googleProduct = storeProduct.googleProduct
+        if (googleProduct == null) {
+            val errorMessage = "Product must be a Google Product."
+            errorLog(errorMessage)
+            purchasesUpdatedListener?.onPurchasesFailedToUpdate(
+                PurchasesError(PurchasesErrorCode.UnknownError, errorMessage)
+            )
+            return
```

Comment:
Moved the validation here... If we consider it only one check, I think it's ok to do it here. I could extract it also to a different function, but I think this should be fine.